### ISSUE TITLE
Help: what is your question?

### ISSUE THREAD

**Issue by hongcui** [biosemantics/etc-site#302]
So some of our Help will initially be empty, as we don't know what to put there. 
I wonder if it would be a good idea to say on those help pages "there is no help information for this page at this time, what do you need help with?"  and ask the user what they need to have in this help page. We could include a text box on those pages and save user input to a database table.

**Comment by jocelynpender** [comment]
I agree with this. I was trying to click on Help because of my "Input directory is not valid. Please select a valid input directory" error. I was unsure if it was a browser issue or if the page didn't yet exist.
